src/store/wizardSlice.ts

```diff
@@ -378,6 +385,11 @@ export const selectUserSshKeyByIndex =
     return state.wizard.users[userIndex]?.ssh_key;
   };
 
+export const selectUserAdministrator =
+  (userIndex: number) => (state: RootState) => {
+    return state.wizard.users[userIndex]?.isAdministrator;
```

Comment:
Do we need optional chaining here?

Reply:
you correct, we define the user without undifined value